plaso/parsers/winfirewall.py

```diff
@@ -224,7 +224,7 @@ def VerifyStructure(self, parser_mediator, line):
     """
     # TODO: Examine other versions of the file format and if this parser should
     # support them.
-    return line == '#Version: 1.5'
+    return '#Version: 1.5' in line
```

Comment:
Why change this?

Reply:
Because of the whitespace matching. Changed this to just have the newline.